UPGRADE-2.0.md

```diff
@@ -1,6 +1,36 @@
 UPGRADE FROM 1.x to 2.0
 =======================
 
+### SonataAdmin Support
+
+ * The Admin extension to edit seo metadata was moved into `symfony-cmf/sonata-admin-integration`.
+   The change of the namespace is handled in the service definition. So you have to change the usage of the
```

Comment:
maybe say "With the move, the admin extension service names also changed. If you are using the `cmf_seo.admin_extension` service, you need to adjust your configuration."